app/views/account/fragments/suspensions.scala.html

```diff
@@ -22,7 +26,18 @@
             <p class="scheduled-suspensions__help">
                 Please contact Customer Services on: <a href="@ContactCentreOps.hrefTelNumber(UK)">@ContactCentreOps.directLine(UK)</a> or email <a href="@ContactCentreOps.hrefMailto">@ContactCentreOps.email</a> if you wish to cancel or amend any of these.
             </p>
-    </div>
 } else {
-    You have no suspensions currently lined up.
+    <p>You have no suspensions currently lined up.</p>
+}
+@if(maybeEmail.isDefined) {
```

Comment:
might be worth a comment to explain why this `maybeEmail` is the decider (I know it's based on sign-in it might just be worth making that more explicit) or additional well-named variable instead